### ISSUE TITLE
Aurora stopped working after Win 10 "Creators Update" (Version 1703)

### ISSUE THREAD

**Issue by rambolger** [antonpup/Aurora#520]
### What are you experiencing an issue with?

- [x] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [ ] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [x] Crash
- [ ] Performance problems
- [ ] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Actual behavior:
Aurora crashes shortly after appearing in system tray for about 3-4 seconds. Keyboard lighting presets will load for a short time and then disappear as soon as Aurora crashes. Aurora UI cannot be opened at all.

### How often does this problem occur?

Every time.

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1.  Upgrade your Windows 10 using [Microsoft Upgrade Tool](https://www.microsoft.com/en-gb/software-download/windows10)
2. Aurora will cease to work.

### What is the version of Aurora you are experiencing this issue with?

My version is 0.6.0b

**Comment by drkpie** [comment]
Happens to me after updating as well, I noticed this after my keyboard lighting profile kept switching back to the one I have set in CUE.

**Comment by WSVizor** [comment]
Yes, i also have the same problem

**Comment by Nieroth** [comment]
Exact same problem for me.
Manually update to 1703 using the upgrade tool and now Aurora instantly crashes everytime.
Still does it after reinstall of Aurora.

**Comment by antonpup** [comment]
I haven't installed the update yet, I will test it when I do.

Could you guys provide me with any kind of logs from Aurora?

**Comment by drkpie** [comment]
[2017_07_04.zip](https://github.com/antonpup/Aurora/files/906724/2017_07_04.zip)

The log file should be in that zip, I hope it helps.

**Comment by Nieroth** [comment]
https://pastebin.com/tu9VwSHX 
Using a Logitech g810

**Comment by WSVizor** [comment]
[2017_07_04.zip](https://github.com/antonpup/Aurora/files/906800/2017_07_04.zip)

**Comment by antonpup** [comment]
Dev version seems to run fine with creators update. Make sure that windows defender isn't blocking Aurora due to being downloaded from internet.

**Comment by drkpie** [comment]
Odd, I have defender off and my AV (Malwarebytes) isn't blocking anything. Neither is my firewall.

**Comment by 1n9i9c7om** [comment]
Can confirm that the release build isn't working on the Creators update right now. It simply crashes a few seconds after starting it, the GUI isn't showing up at all.

I might try to compile the dev branch later today, but I don't really feel like doing it right now. :P

**Comment by antonpup** [comment]
Sorry to inform you, but current release works for me under creator's update. I downloaded the latest version from aurora's website. Since that build, I have reinstalled Windows, so the two machines are "different", so it won't recognize it being built on the same machine.
![](http://i.imgur.com/YwwgTHA.png)

There's possibly some setting that you guys have on, that I do not. Could it be the smart screen?
